pkg/virtctl/imageupload/imageupload.go

```diff
@@ -687,15 +702,18 @@ func createStorageSpec(client kubecli.KubevirtClient, size, storageClass, access
 		spec.AccessModes = []v1.PersistentVolumeAccessMode{v1.PersistentVolumeAccessMode(accessMode)}
 	}
 
-	if blockVolume {
+	if volumeMode == "block" {
```

Comment:
Nit: maybe use here a `switch`. Do we have a default value or do we leave the VolumeMode empty?

Reply:
We leave it empty so the storageProfile default is used.

Reply:
it is really a super nit, but if you change with a switch you can simply set it empty in the default case.

Reply:
Done :+1: